ngrinder-controller/src/test/java/org/ngrinder/perftest/service/AbstractPerfTestTransactionalTest.java

```diff
@@ -68,7 +71,7 @@ public PerfTest newPerfTest(String testName, Status status, Date scheduledTime)
 		test.setDuration(4000L);
 		test.setAgentCount(1);
 		test.setVuserPerAgent(4);
-		test.setScheduledTime(scheduledTime);
+		test.setScheduledTime((scheduledTime == null) ? null : scheduledTime.toInstant());
```

Comment:
take scheduledTime as Instant from the start.

Reply:
@junoyoon 

Yes, I already tried that but many changes have been made.
So, I didn't apply it to minimize the change because this is only for test code.

Would it be better to change?

Reply:
Do not leave any waist.

Reply:
Fixed